_includes/pbc/all/install-features/202304.0/install-the-cart-prices-feature.md

```diff
@@ -0,0 +1,208 @@
+|---------------------------------------------|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Cart                                        | {{site.version}} | [Cart feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/cart-feature-integration.html)                                            |
+| Prices                                      | {{site.version}} | [Prices feature integration](/docs/pbc/all/price-management/{{site.version}}/install-and-upgrade/install-features/install-the-prices-feature.html)              |
+| Marketplace Product Offer Prices (Optional) | {{site.version}} | [Marketplace Product Offer Prices](/docs/marketplace/dev/feature-integration-guides/{{site.version}}/marketplace-product-offer-prices-feature-integration.html) |
+
+### 1) Install the required modules using Composer
+
+Run the following command(s) to install the required modules:
+
+```bash
+composer require spryker/price-cart-connector:"^6.9.0" --update-with-dependencies
+```
+
+{% info_block warningBox “Verification” %}
+
+Make sure that the following modules have been installed:
+
+| MODULE             | EXPECTED DIRECTORY                   |
+|--------------------|--------------------------------------|
+| PriceCartConnector | vendor/spryker/price-cart-connector  |
+
+{% endinfo_block %}
+
+### 2) Set up configuration
+
+Set up the following configuration:
+
+**src/Pyz/Zed/PriceCartConnector/PriceCartConnectorConfig.php**
+```php
+<?php
+
+namespace Pyz\Zed\PriceCartConnector;
+
+use Spryker\Zed\PriceCartConnector\PriceCartConnectorConfig as SprykerPriceCartConnectorConfig;
+
+class PriceCartConnectorConfig extends SprykerPriceCartConnectorConfig
+{
+    protected const IS_ZERO_PRICE_ENABLED_FOR_CART_ACTIONS = false;
+    
+    /**
+     * @return list<string>
+     */
+    public function getItemFieldsForIdentifier(): array
+    {
+        return array_merge(parent::getItemFieldsForIdentifier(), [
+            ItemTransfer::SKU,
+            ItemTransfer::QUANTITY,
+            ...
+        ]);
+    }
+}
+```
+
+If `IS_ZERO_PRICE_ENABLED_FOR_CART_ACTIONS=false`, while attempting to add the product with zero price to the cart, you get the following message: "Price in selected currency not found for product with sku '%sku%'. Please change the currency or remove product from order.".
+
+The `PriceCartConnectorConfig::getItemFieldsForIdentifier()` allows to set up a list of fields that are used to build items identifiers. Based on generated identifiers, system can recognize duplicate items and perform requests only for unique items.
+
+{% info_block warningBox "Warning" %}
+
+Apply the following changes only if you have the [Marketplace Product Offer Prices](/docs/marketplace/dev/feature-integration-guides/{{site.version}}/marketplace-product-offer-prices-feature-integration.html) feature installed.
```

Comment:
this section looks like a part from another IG

Reply:
Discussed and decided to leave it as it is.